src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Deadline {
+
+    protected String start;
```

Comment:
Consider adding a line in between line 3 and line 4 to separate the variables and the constructor.